### PR TITLE
Origin/develop

### PR DESCRIPTION
**Pull request by Anant-Raman**
Varify and merge

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -16,22 +16,57 @@ android {
...
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
-    testImplementation 'junit:junit:4.12'
+    testImplementation 'junit:junit:4.13.2'
+    implementation 'com.android.support:multidex:1.0.3'
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
 
+    //RETROFIT AND GSON
+    implementation "com.squareup.retrofit2:retrofit:2.9.0"
```
Comment (akhilj33) [inline app/build.gradle:51]: Make dependency class separate to maintain reusability and ease in maintainability in future

File: .idea/codeStyles/Project.xml [hunk 2]
```diff
@@ -1,6 +1,22 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
     <JetCodeStyleSettings>
+      <option name="PACKAGES_TO_USE_STAR_IMPORTS">
+        <value>
+          <package name="java.util" alias="false" withSubpackages="false" />
```
Comment (akhilj33) [inline .idea/codeStyles/Project.xml]: We dont have to push .idea file in git. So we should add them to .gitignore. Meanwhile remove the current ones

Comment (Anant-Raman) [reply .idea/codeStyles/Project.xml]: Resolved

File: app/build.gradle (hunk 1 above)
Comment (Anant-Raman) [reply app/build.gradle:51]: Will be done later

**Review by akhilj33 — APPROVED** [review]

**Pull request merged**